src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx

```diff
@@ -244,7 +293,15 @@ function getLeakedPasswords({
     },
   ];
 
-  return leakedPasswordsData.find((content) => content.type === dataType);
+  const unresolvedPasswordBreachContent = leakedPasswordsData.find(
+    (content) => content.type === dataType,
+  );
+
+  return {
+    unresolvedPasswordBreachContent,
+    unresolvedPasswordBreach,
+    unresolvedSecurityQuestionsBreach,
+  };
```

Comment:
Suggestion: Let’s just return the content here. The state updates and the `useEffect` hook in `LeakedPasswordsLayout` can be removed if we get `unresolvedPasswordBreach` and `unresolvedSecurityQuestionsBreach` without having to call `getLeakedPasswords` in order to determine the affected email.

See my suggestion here: